### ISSUE TITLE
Any plan to support docker deployment?

### ISSUE THREAD

**Issue by lhzw** [cert-ee/cuckoo3#24]
Any plan to support docker deployment? It'll much easier to deploy it on the other architecture, and so on, thank you.

**Comment by jurjurijur** [comment]
I would support this. It has been quite a challenge getting this to work on a Debian 11 envoirment and have been unable to make it work stably.

**Comment by rkivistik** [comment]
We are currently reviewing our roadmap and once we have a clearer path forward, we will put components that can be dockerized into docker as an alternative to install script(s).
But unfortunately it is still some time away form being realized. 
We have some urgent work to do before we can get to it so we appreciate the patience. Stay tuned for the roadmap in near(ish) future.

This issue will be closed and the request is already in our to be roadmap.

**Comment by cert-ee-raidar** [comment]
Duplicate of #152

**Comment by cert-ee-raidar** [comment]
Closing the issue as this is under appropriate issue now.
